### DISCUSSION TITLE
How can I create a FIFO SQS queue?

### DISCUSSION THREAD

**Discussion by xiaozhong1995** [amzn/ads-advanced-tools-docs#256] [General]
Hello! I'm trying to use the Amazon Marketing Stream to receive any changes to my advertisements. When I use the CloudFormation template (https://github.com/amzn/ads-advanced-tools-docs/blob/main/amazon_marketing_stream/Stream_SQS%20_CF_Template.yaml) to create an SQS queue, it creates a standard SQS queue.

I would like to create a FIFO SQS queue. Where should I update in the template? Can you help me check if there are any syntax errors?

**Comment by kdwivedi13** [comment]
That's a great point, and I appreciate you bringing it up.  However, I wanted to clarify that Amazon Marketing Stream doesn't currently support a first-in, first-out (FIFO) queue. You can find more details about this in our documentation [here](https://advertising.amazon.com/API/docs/en-us/guides/amazon-marketing-stream/onboarding#sqs-console).
